pyproject.toml

```diff
@@ -1,3 +1,77 @@
 [build-system]
-requires = ["setuptools>=40.8.0", "wheel"]
+requires = ["setuptools", "setuptools-scm", "wheel"]
 build-backend = "setuptools.build_meta"
+
+
+[project]
+name = "zarr"
+description = "An implementation of chunked, compressed, N-dimensional arrays for Python"
+readme = { file = "README.md", content-type = "text/markdown" }
+maintainers = [ {name = "Alistair Miles", email = "alimanfoo@googlemail.com"} ]
```

Comment:
Would it make sense to break these over multiple lines as done below with other sections?